Exercise-7/Java11.java

```diff
@@ -0,0 +1,15 @@
+package exercise7;
+
+import java.io.File;
+
+public class Java11 {
+	public static void main(String[] args) {
+		File file = new File("/Users/gokulananth/eclipse-workspace/Java-Excercise/src/day1");
+		if (file.isDirectory()) {
```

Comment:
It's good to add exception handling here. Eg: If the specified directory does not exist or is not accessible